src/django/api/views/facility/facilities_view_set.py

```
@@ -1411,31 +1449,31 @@ def promote(self, request, pk=None):
-                raise BadRequestException('Match is not to facility')
+                raise BadRequestException("Match is not to facility")
 
             if facility.created_from.id == match.facility_list_item.id:
-                raise BadRequestException('Facility is created from item.')
+                raise BadRequestException("Facility is created from item.")
 
             previous_created_from_id = facility.created_from.id
 
             item = match.facility_list_item.id
             if match.facility_list_item.source.facility_list:
                 list = match.facility_list_item.source.facility_list.id
```

Ruff (0.15.6)

[error] Variable `list` is shadowing a Python builtin

(A001)